modules/zivid/experimental/projection.py

```diff
@@ -0,0 +1,147 @@
+"""Module for experimental projection features. This API may change in the future."""
+
+import _zivid
+from zivid.frame_2d import Frame2D
+from zivid.settings_2d import Settings2D, _to_internal_settings2d
+
+
+class ProjectedImage:
+    """A handle to a 2D image being displayed on a Zivid camera's projector.
+
+    The image projection will stop either when the instance is destroyed, when the stop() method is called,
+    or when exiting the "with" block if used as a context manager.
+    """
+
+    def __init__(self, impl):
+        """Initialize ProjectedImage wrapper.
+
+        This constructor is only used internally, and should not be called by the end-user.
+
+        Args:
+            impl: Reference to internal/back-end instance.
+
+        Raises:
+            TypeError: If argument does not match the expected internal class.
+        """
+        if isinstance(impl, _zivid.ProjectedImage):
+            self.__impl = impl
+        else:
+            raise TypeError(
+                "Unsupported type for argument impl. Got {}, expected {}.".format(
+                    type(impl), type(_zivid.ProjectedImage)
+                )
+            )
+
+    def __str__(self):
+        return str(self.__impl)
+
+    def capture(self, settings2d):
+        """Capture a single 2D frame without stopping the ongoing image projection.
+
+        This function can only be used with a zero-brightness 2D capture, otherwise it
+        will interfere with the projected image. An exception will be thrown if settings
+        contains brightness > 0 or if brightness is not set at all.
```

Comment:
This is also slightly out of sync with C++ version, fex C++ includes "This method returns right after the acquisition of the image is complete." Can you also check the other docstrings also, and either tweak them in the C++ or here so we get them the same?

Reply:
Done, where it makes sense. The Python docstrings can't always be identical because "scope" is replaced with context manager stuff, etc.